articles/azure-maps/migrate-from-google-maps.md

```diff
@@ -31,7 +31,7 @@ If you don't have an Azure subscription, create a [free account] before you begi
 
 ## Azure Maps platform overview
 
-Azure Maps provides developers from all industries powerful geospatial capabilities. The capabilities are packed with regularly updated map data to provide geographic context for web, and mobile applications. Azure Maps has an Azure One API compliant set of REST APIs. The REST APIs offer Maps Rendering, Search, Routing, Traffic, Time Zones, Geolocation, Geofencing, Map Data, and Weather. Operations are accompanied by both Web and Android SDKs to make development easy, flexible, and portable across multiple platforms.
```

Comment:
Can we rephrase the last sentence <Operations are accompanied by both Web and Android SDKs to make development easy, flexible, and portable across multiple platforms.> without Android SDK? I think the last sentence still be worthy to be mentioned since it emphasizes the Azure Maps Web SDK's benefits or advantage besides of Azure Maps APIs.

Reply:
That is perfect, thank you